docs/SMARTonFHIR.md

```diff
@@ -0,0 +1,35 @@
+[SMART on FHIR](https://docs.smarthealthit.org/) is a set of open specifications to integrate partner applications with FHIR servers and electronic medical records systems that have FHIR interfaces. 
+
+There is a tutorial on the managed service documentation that describes how to use the proxy to enable SMART on FHIR applications with the Azure API for FHIR. You can follow the [steps](https://docs.microsoft.com/azure/healthcare-apis/use-smart-on-fhir-proxy) for the open-source FHIR Server as well but need to configure a few settings in a different way first. 
```

Comment:
>can [](start = 163, length = 3)

Maybe verb `must` instead of `can` ? `Can` implies (at least for my non-native skills) what I don't have to do it general, I have ability to do, but it doesn't impose necessity.
Or if `must` is too imperative for your language taste buds, maybe `Please follow this steps ... along with some changes describe below.`

Reply:
I reworded this whole piece.